index.yaml

```diff
@@ -73,6 +73,13 @@ distributions:
     source_builds:
       default: melodic/source-build.yaml
       ds: melodic/source-stretch-build.yaml
+  noetic:
+    ci_builds:
+      bionic-python2: noetic/ci-bionic-python2.yaml
```

Comment:
Since noetic is py3 only, will this job be broken when a repository chooses to make a noetic branch that only supports python 3?

Reply:
Yes, that would be the case if we that repo was added to the Noetic rosdistro and listed as a repo name in the CI build file.